evap/grades/templates/grades_semester_view.html

```diff
@@ -108,8 +108,12 @@ <h3 class="col-8 mb-0">
     {% include 'confirmation_modal.html' with modal_id='confirmNouploadModal' title=title question=question action_text=action_text btn_type='primary' %}
     <script type="text/javascript">
         function confirmNouploadModalAction(dataId) {
-            fetch("{% url 'grades:toggle_no_grades' %}", {
-                body: new URLSearchParams({course_id: dataId}),
+            confirmUploadModalAction(dataId, true);
+        }
+
+        function confirmUploadModalAction(dataId, status) {
```

Comment:
How do you feel about renaming this to something like `updateGradeStatusOfCourse(courseId, status)`?